hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip796/Hip796Verbs.java

```diff
@@ -0,0 +1,324 @@
+
+    // --- Token definition factories ---
+
+    /**
+     * Creates a non-fungible token with the given features and related entities and the default name
+     * ({@code TOKEN_UNDER_TEST}).
+     *
+     * @param features the features to use
+     * @return a {@link TokenDefOperation} that can be used to define the token
+     */
+    public static TokenDefOperation nonFungibleTokenWithFeatures(@NonNull final TokenFeature... features) {
+        return nonFungibleTokenWithFeatures(TOKEN_UNDER_TEST, features);
+    }
+
+    /**
+     * Creates a non-fungible token with the given features and related entities and the given name.
+     *
+     * @param token the token to create
+     * @param features the features to use
+     * @return a {@link TokenDefOperation} that can be used to define the token
+     */
+    public static TokenDefOperation nonFungibleTokenWithFeatures(
+            @NonNull final String token, @NonNull final TokenFeature... features) {
+        // It never makes sense to create a non-fungible token without a supply key
+        final var featuresWithSupplyKey = Stream.concat(
+                        Stream.of(TokenFeature.SUPPLY_MANAGEMENT), Arrays.stream(features))
+                .collect(toCollection(() -> EnumSet.noneOf(TokenFeature.class)))
+                .toArray(TokenFeature[]::new);
+        return tokenWithFeatures(token, TokenType.NON_FUNGIBLE_UNIQUE, featuresWithSupplyKey);
+    }
+
+    /**
+     * Creates a fungible token with the given features and related entities and the default name
+     * ({@code TOKEN_UNDER_TEST}).
+     *
+     * @param features the features to use
+     * @return a {@link TokenDefOperation} that can be used to define the token
+     */
+    public static TokenDefOperation fungibleTokenWithFeatures(@NonNull final TokenFeature... features) {
+        return fungibleTokenWithFeatures(TOKEN_UNDER_TEST, features);
+    }
+
+    /**
+     * Creates a fungible token with the given features and related entities and the given name.
+     *
+     * @param token the token to create
+     * @param features the features to use
+     * @return a {@link TokenDefOperation} that can be used to define the token
+     */
+    public static TokenDefOperation fungibleTokenWithFeatures(
+            @NonNull final String token, @NonNull final TokenFeature... features) {
+        return tokenWithFeatures(token, TokenType.FUNGIBLE_COMMON, features);
+    }
+
+    // --- Lock management verbs ---
+    /**
+     * Locks the given amount of units of the given token in the given account.
+     *
+     * @param account the account to lock the units in
+     * @param token the token to lock the units of
+     * @param amount the amount of units to lock
+     * @return a {@link HapiTxnOp} that can be used to lock the units
+     */
+    public static HapiTxnOp lockUnits(@NonNull final String account, @NonNull final String token, final long amount) {
```

Comment:
I like the format of keeping everything under `hip796` folder 💯

Reply:
Thx @Neeharika-Sompalli 💯